Rubberduck.SourceControl/GitProvider.cs

```diff
@@ -289,7 +291,7 @@ public override void Commit(string message)
                 //The default behavior of LibGit2Sharp.Repo.Commit is to throw an exception if no signature is found,
                 // but BuildSignature() does not throw if a signature is not found, it returns "unknown" instead.
                 // so we pass a signature that won't throw along to the commit.
-                _repo.Commit(message, GetSignature());
+                _repo.Commit(message, GetSignature(), GetSignature());
```

Comment:
To be honest that line threw me for a loop because I thought this was a duplicated code but it's in fact using same signature to `author` and to `committer`.  I would suggest that you create 2 variables named `author` and `committer` as illustrated in [this sample](https://github.com/libgit2/libgit2sharp/wiki/git-commit) to help explain to other people what it is actually.

Reply:
actually ... C#7 does have named arguments. That would accordingly look as follows:

    _repo.Commit(message, author: GetSignature(), committer: GetSignature());

Reply:
Yeah, this seems fair enough in terms of readability. Named arguments are perfect here.